integrations/travisci.rst

```diff
@@ -1,90 +1,83 @@
+It is free for OSS projects, and offers an easy integration with GitHub, so builds can be automatically
+fired in Travis-CI after a :command:`git push` to GitHub.
 
 You can use Travis-CI both for:
 
-- Building and testing your project, which manages dependencies with Conan, and probably a conanfile.txt file
-- Building and testing conan binary packages for a given conan package recipe (with a conanfile.py)
-
-
-Building and testing your project
-------------------------------------
-
-We are going to use an example with GTest package now, with **Travis CI** support to run the tests.
-
-
-Clone the project from github:
-
-
-.. code-block:: bash
+- Building and testing your project, which manages dependencies with Conan, and probably a conanfile.txt file.
+- Building and testing Conan binary packages for a given Conan package recipe (with a *conanfile.py*).
 
-   $ git clone https://github.com/lasote/conan-gtest-example
+Installing dependencies and building your project
+-------------------------------------------------
 
+A very common use case is to build your project after Conan takes care of installing your dependencies. Doing this process in Travis CI is
+quite convenient as you can do it with :command:`conan install`.
 
-Create a ``.travis.yml`` file and paste this code in it:
-
+To enable **Travis CI** support, you need to create a *.travis.yml* file and paste this code in it:
 
 .. code-block:: text
 
-	language: cpp
-	compiler:
-	- gcc
-	install:
-	# Upgrade GCC
-	- sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
-	- sudo apt-get update -qq
-	- sudo apt-get install -qq g++-4.9
-	- sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.9 60 --slave /usr/bin/g++ g++ /usr/bin/g++-4.9
-
-	# Install conan
-	- pip install conan
-	# Automatic detection of your arch, compiler, etc.
-	- conan user
-
-	script:
-	# Download dependencies, build, test and create package
-	- conan create . user/channel
-
-
-Travis will install the **conan** tool and will execute the **conan install** command.
-Then, the **script** section creates the build folder, compiles the project with **cmake** and runs the **tests**.
-
-
-Creating, testing and uploading conan binary packages
--------------------------------------------------------
-
-You can use Travis to automate the building of binary packages, which will be created in the
-cloud after pushing to Github. You can probably setup your own way, but conan has some utilities to help in the process.
+    sudo: false
+    language: cpp
+    compiler:
+      - gcc
+    addons:
+      apt:
+        sources:
+          - ubuntu-toolchain-r-test
+        packages:
+          - g++-6
+    install:
+      - "[ $CXX = g++ ] && export CXX=g++-6 || true"
+    # Install conan
+      - pip install conan
+    # Automatic detection of your arch, compiler, etc.
+      - conan user
+    script:
+    # Download dependencies and build project
+      - conan install .
+      - make
```

Comment:
Add a comment here, or maybe the more canonical way: `cmake . -G "Unix makefiles" && make`
And then a comment (or a command) related to _"run your test suite"_

Reply:
you are assuming the usage of CMake in this case and I would like to not focus the attention on that. I can change it and add some comments

Reply:
yes, I know it, but I prefer a fully working example. It is not necessary to talk about CMake, but if this file is valid for a project using CMake (copy-paste-works) it would be better.

Reply:
Done